astropy_helpers/sphinx/conf.py

```diff
@@ -135,15 +135,11 @@ def check_sphinx_version(expected_version):
     'sphinx.ext.coverage',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.viewcode',
-    'astropy_helpers.sphinx.ext.numpydoc',
-    'astropy_helpers.sphinx.ext.astropyautosummary',
-    'astropy_helpers.sphinx.ext.autodoc_enhancements',
-    'astropy_helpers.sphinx.ext.automodsumm',
```

Comment:
While ``automodsumm`` is getting included via ``automodapi`` and ``autodoc_enhancements`` via ``automodsum``, I think it's preferable to list them here to be more explicit.